### DISCUSSION TITLE
Buffer keymaps

### DISCUSSION THREAD

**Discussion by vinicius507** [LazyVim/LazyVim#388] [Q&A] [answered]
Is there any recommended way of adding plugins buffer keymaps?

I'm thinking of using autocmds, but I'd like to know if there are ways to do it built into the LazySpec.

Thanks in advance.

**Comment by folke** [comment, marked as answer]
Nope, just add them as without LazyVim, so autocmds indeed

**Reply by vinicius507** [reply]
Thanks!
